scripts/gen-nncp.sh

```diff
@@ -323,6 +323,9 @@ EOF_CAT
         cat >> ${DEPLOY_DIR}/${WORKER}_nncp.yaml <<EOF_CAT
       ipv6:
         enabled: false
+EOF_CAT
+    fi
+    cat >> ${DEPLOY_DIR}/${WORKER}_nncp.yaml <<EOF_CAT
       mtu: ${INTERFACE_MTU}
```

Comment:
may be better to move this block to L295, and drop duplicated mtu and state?

Reply:
Makes sense, Done.